stl/inc/experimental/vector

```diff
@@ -9,7 +9,7 @@
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
 
-#include <experimental/xutility>

```

Comment:
It looks like you're no longer using `<experimental/xutility>` because all of its machinery is being moved into `std` (currently `<algorithm>`, soon `<xmemory>` as other comments have mentioned). In that case, `<experimental/xutility>` should simply be deleted.